Makefile

```diff
@@ -44,10 +44,10 @@ coverage:
 	coverage report -m
 
 docs:
-	sphinx-apidoc -o docs/ eggplant
+	#sphinx-apidoc -o docs/ eggplant
 	$(MAKE) -C docs clean
 	$(MAKE) -C docs html
-	open docs/_build/html/index.html
+	xdg-open docs/_build/html/index.html
```

Comment:
Aaaah, that way :) Are you saying there are devs on OSX!? :D I kept getting an error on `open`... I'll just remove the whole line yes.